### PR TITLE
Enable accessibility in CKLabelComponent by default

### PR DESCRIPTION
**Pull request by ocrickard**
Enables accessibility by default in CKLabelComponent, and provides an example on how to do more advanced accessibility setups for CKTextComponent.

Should fix https://github.com/facebook/componentkit/issues/127

### CODE REVIEW

File: ComponentTextKit/CKLabelComponent.mm [hunk 1]
```diff
@@ -17,12 +17,12 @@ @implementation CKLabelComponent
 + (instancetype)newWithLabelAttributes:(const CKLabelAttributes &)attributes
                         viewAttributes:(const CKViewComponentAttributeValueMap &)viewAttributes
 {
-  CKViewComponentAttributeValueMap copiedMap = viewAttributes;
+  const CKViewComponentAttributeValueMap copiedMap = viewAttributes;
```
Comment (adamjernst) [inline ComponentTextKit/CKLabelComponent.mm]: Don't add `const` here or the copied map can't be moved (I think).

Comment (adamjernst) [inline ComponentTextKit/CKLabelComponent.mm]: Don't add const here or the copied map can't be moved (I think).

Comment (ocrickard) [inline ComponentTextKit/CKLabelComponent.mm]: Yeah, good point this degrades to a copy instead of a move if it's const.

**Comment by ocrickard** [comment]
Updated diff.

**Pull request merged**